src/components/bottom-control-buttons/components/PlaybackRateButton.tsx

```diff
@@ -24,7 +24,7 @@ export const PlaybackRateButton: FC<PlaybackRateButtonProps> = ({
 
 	const handleClick = () => {
 		// Gets the next value of playback rate, otherwise get first one
-		const playbackLength = PLAYBACK_RATES.length;
+		const playbackLength = PLAYBACK_RATES.length - 1;
```

Comment:
Could you move this into a separate hook and add unit test? I think that's the kind of business logic that tends to break on the edges.
For inspiration: The PlaylistPage has similar kind of rotation logic. Here are the unit tests for it: https://github.com/Collaborne/carrot-app-ui/blob/main/src/regions/playlist/hooks/__tests__/useRotatingArray.spec.ts